Gems/Multiplayer/Code/Include/Multiplayer/MultiplayerTypes.h

```diff
@@ -249,14 +250,17 @@ namespace Multiplayer
         }
 
         const auto potentialNetworkSpawnable = reinterpret_cast<AZ::Data::Asset<AzFramework::Spawnable>*>(newValue);
-        if (!potentialNetworkSpawnable->GetHint().ends_with(NetworkSpawnableFileExtension))
+        const auto& hint = potentialNetworkSpawnable->GetHint();
```

Comment:
are hints removed in the final release game build .pak bundle output? Is that why you allowed `hint.empty()`?

Reply:
No, the hint.empty() is because you should be allowed to clear the reference on the Simple Network Player Spawner. Without this change, the following occurs in the Editor:
1. Add a Simple Network Player Spawner component to the level, which by default has an empty hint / asset.
2. Set the spawner asset to something valid.
3. Try to clear the asset by pressing the X. It refuses to let you clear the asset, it instead prints an error message.

With this change, the X works, and you can clear the asset.

It's just a small workflow improvement, it's a little bit annoying to have an asset set that you can't clear without deleting the component.

Reply:
Nice find/fix!